scene/gui/popup_menu.cpp

```diff
@@ -1015,6 +1006,8 @@ void PopupMenu::_notification(int p_what) {
 		case NOTIFICATION_THEME_CHANGED:
 		case Control::NOTIFICATION_LAYOUT_DIRECTION_CHANGED:
 		case NOTIFICATION_TRANSLATION_CHANGED: {
+			scroll_container->add_theme_style_override("panel", theme_cache.panel_style);
```

Comment:
I just wanted to set the `ScrollContainer` panel style in NOTIFICATION_THEME_CHANGED, but the three of them are linked, and it will result in creating and extra function or duplicating code.

Reply:
You can use `[[falthrough]]`.